exoctk/exoctk_app/templates/base.html

```diff
@@ -85,6 +85,7 @@
                         </ul>
                         </li>
                         <li><a href="limb_darkening">Limb Darkening Calculator</a></li>
+                        <li><a href="phase_contraint">Phase Constraint Calculator</a></li>
```

Comment:
did you miss the `s` in `phase_contraint`?

Reply:
Whoops!  Good catch!